debian/install

```diff
@@ -0,0 +1,6 @@
+b2g/* /opt/b2g/b2g/
```

Comment:
Do we really need that list of `b2g`? that being said, that's not the point of this PR, so we could refactor that later.